README.md

```diff
@@ -106,6 +106,12 @@ Available options:
   --secrets-file FILENAME  config file specifying which secrets to request
   --no-connect-tls         don't use TLS when connecting to Vault
   --no-inherit-env         don't merge the current environment with the secret
+  --retry-base-delay-milliseconds MILLISECONDS
+                           base delay for vault connection retrying. Defaults to
+                           40ms because, in testing, we found out that fetching
+                           50 secrets takes roughly 200 milliseconds
```

Comment:
Just out of curiosity, how did we reach the conclusion that 40ms is the right base delay based on the fact that 50 secrets take 200 ms?

Reply:
I remember deciding this with @ruuda, but I don't know how we came to that number anymore and I can't seem to find the old discussion. (Maybe Ruud knows more details)

I remember it being a bit hand-wavy: "this normally takes 200ms to fetch 50 secrets, so if we wait a bit we can hopefully keep this reasonably snappy without running out of retry options or overloading Vault". (Our internal tooling had around that number of secrets at the time).

Reply:
If you have a better way to think up a number, we can reconsider. Then again, it has been working pretty well for us